src/pages/cookie-policy/cookie-policy.styles.js

```diff
@@ -0,0 +1,37 @@
+export const styles = {
+  container: {
+    color: 'primary.900'
+  },
+  sectionTitle: {
+    '&:first-of-type': {
+      backgroundColor: 'basic.grey',
+      m: '30px 0 70px 0',
+      py: '25px',
+      borderRadius: '20px'
+    }
+  },
+  itemsContainer: {
+    m: '0 auto 40px',
+    maxWidth: '744px'
+  },
+  wrapper: {
+    textAlign: 'start'
+  },
+  title: {
+    mb: '30px'
+  },
+  titleWithDot: {
```

Comment:
maybe will be better if you use <li> tag for that

Reply:
I think it's better to just pass the text for the title in titleWithDescription component and create a dot by css, than use extra tags and maybe change the component.